site/docs/try/configuration.md

```diff
@@ -234,30 +231,22 @@ Metrics are published using prometheus and can be collected via standard methods
 Since Nessie 0.46.0, traces are published using OpenTelemetry. See [Using
 OpenTelemetry](https://quarkus.io/guides/opentelemetry) in the Quarkus documentation.
 
-In order for the server to publish its traces, the
-`quarkus.otel.exporter.otlp.endpoint` property _must_ be set. Its value must be a
+In order for the server to enable OpenTelemetry and publish its traces, the
+`quarkus.otel.exporter.otlp.traces.endpoint` property _must_ be defined. Its value must be a
 valid collector endpoint URL, with either `http://` or `https://` scheme. The collector must talk
 the OpenTelemetry protocol (OTLP) and the port must be its gRPC port (by default 4317), e.g.
-"http://otlp-collector:4317".
+"http://otlp-collector:4317". _If this property is not set, the server will not publish traces._
 
-Alternatively, it's possible to disable opentelemetry completely at runtime by setting the following 
+Alternatively, it's possible to forcibly disable OpenTelemetry at runtime by setting the following 
 property: `quarkus.otel.sdk.disabled=true`.
 
 #### Troubleshooting traces
 
 If the server is unable to publish traces, check first for a log warning message like the following:
 
 ```
-WARN  [io.qua.ope.run.exp.otl.LateBoundBatchSpanProcessor] (vert.x-eventloop-thread-5) No BatchSpanProcessor delegate specified, no action taken.
```

Comment:
My understanding is that this warning cannot happen anymore, because now the endpoint URL is by default set to `http://localhost:4317` in all profiles, and not only in `dev`.